### PR TITLE
Add Share Link

### PR DESCRIPTION
**Pull request by jperegrino75**
Added "Sharing Tools" Link to more menu

### CODE REVIEW

File: src/lib/useNavigationItems.tsx [hunk 1]
```diff
@@ -470,6 +470,16 @@ export function useNavigationItems(): INavigationItem[] {
 						defaultMessage: 'Get Help',
 					}),
 				},
+				{
+					key: 'sharingtools',
+					href: 'http://share.audioverse.org/',
```
Comment (alangumer) [inline src/lib/useNavigationItems.tsx:475]: Should be https and there is no need for the slash at the end.

**Review by alangumer — CHANGES_REQUESTED** [review]
The translations are missing.

File: src/lib/useNavigationItems.tsx (hunk 1 above)
Comment (jperegrino75) [reply src/lib/useNavigationItems.tsx]: done and committed

**Review by alangumer — APPROVED** [review]

**Comment by jperegrino75** [comment]
> The translations are missing.

we will keep the link to English in all languages as this is the way the other pages are currently.

**Comment by jperegrino75** [comment]
@alangumer I resolved the conflicts by removing the code. This was for internationalization of the Sharing Link but since this is the same for all languages; I just removed the code. Hope I did it in the right way :-)

**Pull request merged**